docs/sources/flow/reference/cli/fmt.md

```diff
@@ -6,7 +6,8 @@ aliases:
 canonical: https://grafana.com/docs/agent/latest/flow/reference/cli/fmt/
 description: The `fmt` command formats a Grafana Agent configuration file.
 menuTitle: fmt
-title: fmt command
+title: `fmt`` command
```

Comment:
```suggestion
title: fmt command
```

You can't use Markdown in the titles

Reply:
Actually that suggestion looks incorrect given the convention you've adopted with the other commands of "The `<NAME>` command"

Reply:
Yeah this is one I fixed local and forgot to push it yesterday :-P It was late.  Fixed to match the other CLI topic titles.